decorator.rb

```diff
@@ -0,0 +1,37 @@
+require './nameable'
+require './person'
+
+class Decorator < Nameable
+  attr_accessor :nameable
+
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
+
+  def correct_name
+    @nameable.correct_name
+  end
+end
+
+class CapitalizeDecorator < Decorator
```

Comment:
### Optional
- [ ] It would be nice if we create a separate file for each of the class